Fixed issue with InventoryMenu and added Loan Menu

Zyra915 opened the pull request:
NOT functional, simply the UI. Bold fonts have issues

Zyra915 committed:
Fixed bug where InventoryItems were scaled wrong

Zyra915 committed:
Loans desk standardized

Did not add ANY functionality

Zyra915 committed:
Fixed issue with closing loanMenu

zachj112 committed:
Removed ZachJTesting added incomplete refs for loan menu

zachj112 reviewed (approved):
Loan Menu UI looks good thanks. Will add logic

Zyra915 committed:
Added boxes to outside of store

Boxes appear per 20 items in the inventory

Zyra915 committed:
Added various upgrades and hires, fixed order of employees

Fixed IDs for employees in CustomerManager.
Added functionality for upgrades 7 - 10
Added functionality for hires 6, 7, 8, 12

Zyra915 committed:
Fixed some IDs

Zyra915 committed:
Merge branch 'main' into ZachMDev_3-26

Zyra915 committed:
Merged Main into branch

Fixed Merge issues, changed some scaling issues

Zyra915 committed:
Moved Tutorial Storage to JSON database
